contracts-abi/config/mainnet.go

```diff
@@ -0,0 +1,17 @@
+package config
+
+var MainnetContracts = Contracts{
```

Comment:
Calling these "mainnet contracts" is confusing imo. They're not deployed to mainnet ethereum (L1), they're deployed on mev-commit chain